<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>835</width>
    <height>705</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">VVHD cylinder</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../mainres.qrc">
    <normaloff>:/icons/app.svg</normaloff>:/icons/app.svg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="PropertiesLayout">
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QGridLayout" name="BasicsLayout">
          <item row="3" column="0">
           <widget class="QLabel" name="ReLabel">
            <property name="text">
             <string>Reynolds number
(by radius)</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="dtLabel">
            <property name="text">
             <string>Time step</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="InfSpeedXLabel">
            <property name="text">
             <string>Infinity speed X</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="InfSpeedYLabel">
            <property name="text">
             <string>Infinity speed Y</string>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="RotationLabel">
            <property name="text">
             <string>Rotation speed</string>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="BodyVortsLabel">
            <property name="text">
             <string>Body vortexes</string>
            </property>
           </widget>
          </item>
          <item row="9" column="0">
           <widget class="QLabel" name="HeatEnabledLabel">
            <property name="text">
             <string>Heat enabled</string>
            </property>
           </widget>
          </item>
          <item row="9" column="1">
           <widget class="QCheckBox" name="HeatEnabledEdit">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="BasicsLabel">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Basics</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="FileNameLabel">
            <property name="text">
             <string>File name</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="1">
           <layout class="QGridLayout" name="fileNameLayout">
            <item row="0" column="0">
             <widget class="QSuperEdit" name="FileNameEdit">
              <property name="spareButtonEnabled" stdset="0">
               <bool>true</bool>
              </property>
              <property name="editable" stdset="0">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="3" column="1">
           <layout class="QGridLayout" name="ReLayout">
            <item row="0" column="0">
             <widget class="QSuperEdit" name="ReEdit">
              <property name="spareButtonEnabled" stdset="0">
               <bool>false</bool>
              </property>
              <property name="editable" stdset="0">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="4" column="1">
           <layout class="QGridLayout" name="dtLayout">
            <item row="0" column="0">
             <widget class="QSuperEdit" name="dtEdit">
              <property name="spareButtonEnabled" stdset="0">
               <bool>false</bool>
              </property>
              <property name="editable" stdset="0">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="5" column="1">
           <layout class="QGridLayout" name="InfSpeedXLayout">
            <item row="0" column="0">
             <widget class="QSuperEdit" name="InfSpeedXEdit">
              <property name="spareButtonEnabled" stdset="0">
               <bool>false</bool>
              </property>
              <property name="editable" stdset="0">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="6" column="1">
           <layout class="QGridLayout" name="InfSpeedYLayout">
            <item row="0" column="0">
             <widget class="QSuperEdit" name="InfSpeedYEdit">
              <property name="spareButtonEnabled" stdset="0">
               <bool>false</bool>
              </property>
              <property name="editable" stdset="0">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="7" column="1">
           <layout class="QGridLayout" name="RotationLayout">
            <item row="0" column="0">
             <widget class="QSuperEdit" name="RotationEdit">
              <property name="spareButtonEnabled" stdset="0">
               <bool>false</bool>
              </property>
              <property name="editable" stdset="0">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="8" column="1">
           <layout class="QGridLayout" name="BodyVortsLayout">
            <item row="0" column="0">
             <widget class="QSuperEdit" name="BodyVortsEdit">
              <property name="spareButtonEnabled" stdset="0">
               <bool>false</bool>
              </property>
              <property name="editable" stdset="0">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QGridLayout" name="AdvancedLayout">
          <item row="3" column="0">
           <widget class="QLabel" name="MinNodeSizeLabel">
            <property name="text">
             <string>Minimum node sise</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="MinGLabel">
            <property name="text">
             <string>Minimum circulation</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="ConvEpsLabel">
            <property name="text">
             <string>Convective epsilon</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="MergeEpsLabel">
            <property name="text">
             <string>Merge epsilon</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="TreeFarLabel">
            <property name="text">
             <string>Tree far criteria</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="AdvancedLabel">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Advanced</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <layout class="QGridLayout" name="TreeFarLayout">
            <item row="0" column="0">
             <widget class="QSuperEdit" name="TreeFarEdit">
              <property name="spareButtonEnabled" stdset="0">
               <bool>false</bool>
              </property>
              <property name="editable" stdset="0">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="3" column="1">
           <layout class="QGridLayout" name="MinNodeSizeLayout">
            <item row="0" column="0">
             <widget class="QSuperEdit" name="MinNodeSizeEdit">
              <property name="spareButtonEnabled" stdset="0">
               <bool>true</bool>
              </property>
              <property name="editable" stdset="0">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="4" column="1">
           <layout class="QGridLayout" name="MinGLayout">
            <item row="0" column="0">
             <widget class="QSuperEdit" name="MinGEdit">
              <property name="spareButtonEnabled" stdset="0">
               <bool>false</bool>
              </property>
              <property name="editable" stdset="0">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="5" column="1">
           <layout class="QGridLayout" name="ConvEpsLayout">
            <item row="0" column="0">
             <widget class="QSuperEdit" name="ConvEpsEdit">
              <property name="spareButtonEnabled" stdset="0">
               <bool>false</bool>
              </property>
              <property name="editable" stdset="0">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="6" column="1">
           <layout class="QGridLayout" name="MergeEpsLayout">
            <item row="0" column="0">
             <widget class="QSuperEdit" name="MergeEpsEdit">
              <property name="spareButtonEnabled" stdset="0">
               <bool>true</bool>
              </property>
              <property name="editable" stdset="0">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QGridLayout" name="OutputLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="OutputLabel">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Output</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="PrintFreqLabel">
            <property name="text">
             <string>Print frequency</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="PrintFreqEdit">
            <property name="accelerated">
             <bool>true</bool>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>100000</number>
            </property>
            <property name="value">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="ValuesLabel">
            <property name="text">
             <string>Values to save</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="InvisibleEdit">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="frame">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <layout class="QGridLayout" name="ValuesToSaveLabel">
            <item row="0" column="0">
             <widget class="QLineEdit" name="ValuesToSaveEdit">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="frame">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="ValuesToSaveEditButton">
              <property name="minimumSize">
               <size>
                <width>27</width>
                <height>27</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>27</width>
                <height>27</height>
               </size>
              </property>
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="RightBarLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinimumSize</enum>
        </property>
        <item>
         <layout class="QGridLayout" name="ToolLayout">
          <property name="sizeConstraint">
           <enum>QLayout::SetMinAndMaxSize</enum>
          </property>
          <item row="0" column="2">
           <widget class="QPushButton" name="RemovePushButton">
            <property name="minimumSize">
             <size>
              <width>27</width>
              <height>27</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>27</height>
             </size>
            </property>
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="DublicatePushButton">
            <property name="minimumSize">
             <size>
              <width>27</width>
              <height>27</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>27</height>
             </size>
            </property>
            <property name="text">
             <string>d</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="AddPushButton">
            <property name="minimumSize">
             <size>
              <width>27</width>
              <height>27</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>27</height>
             </size>
            </property>
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QDDListWidget" name="listWidget">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="acceptDrops">
           <bool>true</bool>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="dragEnabled">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::ExtendedSelection</enum>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="verticalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
          <property name="horizontalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
          <property name="viewMode">
           <enum>QListView::ListMode</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="SaveButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Save</string>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>835</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPreferences">
    <property name="title">
     <string>Preferences</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuPreferences"/>
   <addaction name="menuHelp"/>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QDDListWidget</class>
   <extends>QListWidget</extends>
   <header>src/qddlistwidget.h</header>
  </customwidget>
  <customwidget>
   <class>QSuperEdit</class>
   <extends>QLineEdit</extends>
   <header>src/qsuperedit.h</header>
   <slots>
    <slot>edit()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>FileNameEdit</tabstop>
  <tabstop>ReEdit</tabstop>
  <tabstop>dtEdit</tabstop>
  <tabstop>InfSpeedXEdit</tabstop>
  <tabstop>InfSpeedYEdit</tabstop>
  <tabstop>RotationEdit</tabstop>
  <tabstop>BodyVortsEdit</tabstop>
  <tabstop>HeatEnabledEdit</tabstop>
  <tabstop>TreeFarEdit</tabstop>
  <tabstop>MinNodeSizeEdit</tabstop>
  <tabstop>MinGEdit</tabstop>
  <tabstop>ConvEpsEdit</tabstop>
  <tabstop>MergeEpsEdit</tabstop>
  <tabstop>InvisibleEdit</tabstop>
  <tabstop>PrintFreqEdit</tabstop>
  <tabstop>ValuesToSaveEdit</tabstop>
  <tabstop>ValuesToSaveEditButton</tabstop>
  <tabstop>AddPushButton</tabstop>
  <tabstop>DublicatePushButton</tabstop>
  <tabstop>RemovePushButton</tabstop>
  <tabstop>listWidget</tabstop>
  <tabstop>SaveButton</tabstop>
 </tabstops>
 <resources>
  <include location="../mainres.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>addFile()</slot>
  <slot>dublicateFile()</slot>
  <slot>removeFile()</slot>
 </slots>
</ui>
